cloudinit/config/cc_power_state_change.py

```diff
@@ -152,7 +156,10 @@ def load_power_state(cfg):
     if not isinstance(pstate, dict):
         raise TypeError("power_state is not a dict.")
 
-    opt_map = {'halt': '-H', 'poweroff': '-P', 'reboot': '-r'}
+    if distro_name == 'alpine':
+        opt_map = {'halt': '', 'poweroff': '', 'reboot': ''}
+    else:
+        opt_map = {'halt': '-H', 'poweroff': '-P', 'reboot': '-r'}
```

Comment:
This change isn't needed. opt_map values are only only used in the non-alpine path.

Reply:
Done